chaos_genius/core/anomaly/processor.py

```diff
@@ -115,15 +143,18 @@ def _predict(self, model: AnomalyModel):
                     self.freq,
                     pred_df=input_data,
                 )
-
                 prediction["y"] = input_data["y"]
-                prediction_with_severity = self._detect_severity(self._detect_anomalies(prediction))
-                pred_series = prediction_with_severity
+
+                pred_series = self._calculate_metrics(
+                    prediction, prediction["y"].mean(), prediction["y"].std()
```

Comment:
The mean here is a single value, but don't we need a mean of last [anomaly-period] days for every timestamp?

Reply:
Not here. This is the condition where we are running anomaly for the first time (refer `if self.last_date is None`). Here everything just happens to the bulk of the data all at once.

If you look at the else condition you'll see it happening for every timestamp.

Reply:
Yes, makes sense